modules/ROOT/pages/mule-wmq-transport-reference.adoc

```diff
@@ -509,7 +525,7 @@ An endpoint to which WMQ messages are sent.
 *Type*: boolean +
 *Required*: no +
 *Default*: none
-|correlationId |A client can use the correlation ID header field to link one message to another. A typical use case is to link a response message with its request message. The CorrelationID must be a 24-byte String. WebSphere pads shorter values with zeroes so that the fixed length is always 24 bytes. Because each message sent by a WMQ provider is assigned a message ID value, it is convenient to link messages via the message ID. All message ID values must start with the 'ID:' prefix.
+|correlationId |A client can use the correlation ID header field to link one message to another. A typical use case is to link a response message with its request message. WMQ uses a 24-byte string for correlation ID. WebSphere pads shorter values with zeroes so that the fixed length is always 24 bytes. Because each message sent by a WMQ provider is assigned a message ID value, it is convenient to link messages via the message ID. By default, all message ID values start with the 'ID:' prefix. To preserve a provided correlation ID, review the <<using-correlationid-provided,using a correlation ID provided>> documentation section..
```

Comment:
```suggestion
|correlationId |A client can use the correlation ID header field to link one message to another. A typical use case is to link a response message with its request message. WMQ uses a 24-byte string for the correlation ID. WebSphere pads shorter values with zeroes so that the fixed length is always 24 bytes. Because each message sent by a WMQ provider is assigned a message ID value, it is convenient to link messages via the message ID. By default, all message ID values start with the `ID:` prefix. To preserve a provided correlation ID, review the <<using-correlationid-provided,using a correlation ID provided>> documentation section..
```